packages/perseus/src/widgets/__tests__/explanation.test.ts

```diff
@@ -41,167 +85,184 @@ describe("Explanation", function () {
         await userEvent.click(screen.getByRole("button"));
 
         // Assert
+        // The only real difference between expanded and not expanded is the
+        //     classes and aria that are applied.
         expect(container).toMatchSnapshot("expanded");
     });
 
-    it("should snapshot for mobile", async () => {
-        // Arrange and Act
-        const {container} = renderQuestion(question1, {
-            isMobile: true,
-        });
+    it("does NOT have a test ID when not in a test environment", async () => {
```

Comment:
**thought**: I mentioned this later on also, but I think it's overkill to remove test ids for prod. To date, I'm not aware of any Khan Academy code that suppresses test IDs in prod, so I think we'd want to talk about it as a team before we start doing that.